tests/unit/track/test_track.py

```diff
@@ -380,3 +382,30 @@ def test_trajectories_missing_output(self, mocker):
             FileNotFoundError, match="TRACK output trajectory file does not exist"
         ):
             tracker.trajectories()
+
+    def test_variable_metadata(self, mocker):
```

Comment:
Apologies for not catching earlier, but spotted we seem to be mixing and matching `unittest mock` with pytest in this file. We should really standardise on pytest.

`unittest mock` used on ll. 74, ll. 78, ll. 96, ll. 97